client/packages/purchasing/src/purchase_order/api/hooks/usePurchaseOrderLine.ts

```diff
@@ -18,6 +18,7 @@ export type DraftPurchaseOrderLine = Omit<
   purchaseOrderId: string;
```

Comment:
Just a thought, though not strictly related to the PR (but we may want a followup?) -- should the "requestedPackSize" default to the item's default pack size?

And if not, I think it should at least default to `1` -- it's weird seeing a pack size of `0` as that's not actually a valid value.

<img width="356" height="147" alt="Image" src="https://github.com/user-attachments/assets/2eccd4c5-f27e-48cb-a578-4746e82e8bc5" />

Reply:
Okay, should probably make a follow-up issue for that.

In the meantime, there's just this little linter error, then we should be good to go:

>   ./open-msupply/client/packages/purchasing/src/purchase_order/DetailView/LineEdit/utils.ts
>   97:7  error  'numberOfPacks' is never reassigned. Use 'const' instead      prefer-const
>   98:7  error  'requestedPackSize' is never reassigned. Use 'const' instead  prefer-const